rpcs3/Emu/Io/usio.cpp

```diff
@@ -128,12 +126,18 @@ void usb_device_usio::load_backup()
 
 void usb_device_usio::save_backup()
 {
-	if (!usio_backup_file && !usio_backup_path.empty() && !usio_backup_file.open(usio_backup_path, fs::create + fs::write + fs::lock))
+	if (!is_used)
+		return;
+
+	fs::file usio_backup_file;
+
+	if (!usio_backup_file.open(usio_backup_path, fs::create + fs::write + fs::lock))
 	{
-		usio_log.error("Failed to create a new USIO Backup file: %s", usio_backup_path);
+		usio_log.error("Failed to save the USIO Backup file: %s", usio_backup_path);
 		return;
 	}
 
+	usio_backup_file.trunc(0);
```

Comment:
You can trunc to write size after the write, it avoids misleading the file system that it can free sectors on existing file.

Reply:
Fixed.